__tests__/app.test.js

```diff
@@ -44,3 +44,33 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:articleId", () => {
+  test("Should return an article object with the relevant properties", () => {
+    return request(app)
+      .get("/api/articles/5")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toMatchObject({
```

Comment:
is there a check you could do here to make sure the correct article has been sent back?